tensorizer/serialization.py

```diff
@@ -719,10 +795,11 @@ def read_tensors(
             yield module_idx, tensor_type, name, tensor.to_tensor()
 
     def read_numpy_arrays(
-        self,
-        filter_func: Optional[Callable[[str], Union[bool, Any]]] = None,
-        num_tensors: int = -1,
-        allow_raw_data: bool = False,
+            self,
+            filter_func: Optional[Callable[[str], Union[bool, Any]]] = None,
+            num_tensors: int = -1,
+            allow_raw_data: bool = False,
+            verify_hash: Union[bool, None] = None,
     ) -> Iterator[Tuple[int, int, str, numpy.ndarray, bool, Optional[str]]]:
```

Comment:
This function needs to have its docstring updated to explain its new `verify_hash` parameter.